Automattic/jetpack
# SSO: only enable WordPress.com invite emails by default on wpcom

> [!NOTE]
> #36498 will need to be merged first; this branch is built against `update/sso-invite-logic`.

## Proposed changes:

The new interface elements introduced in pdDR7T-1mt-p2 are first designed for WordPress.com sites. Let's reflect that in the interface.

**On self-hosted sites**

| **before** | **after** |
|--------|--------|
| <img width="770" alt="Screenshot 2024-03-25 at 12 35 43" src="https://github.com/Automattic/jetpack/assets/426388/5b64ba92-329d-41a9-9878-e809cc931fe9"> | <img width="784" alt="Screenshot 2024-03-25 at 12 34 54" src="https://github.com/Automattic/jetpack/assets/426388/d98df806-55cc-4f16-b25e-c7a2645da2aa"> | 

### Other information:


## Jetpack product discussion

* pdDR7T-1mt-p2

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

Test this on different environments:

- On a self-hosted site
- On a WoA site
- On a WoA site with the WooCommerce plugin active.

* The sites must be connected to WordPress.com.
* Go to Users > Add New
    1. The "Invite user to WordPress.com" checkbox should only be checked on WoA sites, when the WooCommerce plugin is not active.
    2. The "This user is a contractor, freelancer, consultant, or agency." checkbox should only be displayed on WoA sites.
